### PR TITLE
Add project compilation via ccmix and make

### PR DESCRIPTION
**Pull request by MahBoiDeveloper**
This changes will allow track direct files instead of tracking `.mix` archives.

To build project run `Build.bat`.

### CODE REVIEW

**Comment by mbnq** [comment]
Thanks, I'll check it.

**Comment by MahBoiDeveloper** [comment]
> Thanks, I'll check it.

If you have questions - feel free to ask me.

File: tools/ccmix/global mix cache.dat
Comment (Metadorius) [inline tools/ccmix/global mix cache.dat:1]: do you think this file is needed?

File: tools/ccmix/global mix database.dat
Comment (Metadorius) [inline tools/ccmix/global mix database.dat:1]: and this one

Comment (MahBoiDeveloper) [reply tools/ccmix/global mix database.dat:1]: ¯\_(ツ)_/¯

File: tools/ccmix/global mix cache.dat
Comment (MahBoiDeveloper) [reply tools/ccmix/global mix cache.dat:1]: ¯\_(ツ)_/¯

**Pull request merged**